src/components/_inputs/TextField/TextField.styles.ts

```diff
@@ -1,5 +1,24 @@
 import styled from '@emotion/styled'
 
-export const TextInput = styled.input`
+import { sizes } from '@/styles'
+
+export const TextInput = styled.input<{ leftNodeWidth: number; rightNodeWidth: number }>`
   width: 100%;
+  padding-left: ${({ leftNodeWidth }) => (leftNodeWidth ? `${leftNodeWidth}px !important` : 'inherit')};
+  padding-right: ${({ rightNodeWidth }) => (rightNodeWidth ? `${rightNodeWidth}px !important` : 'inherit')};
+`
+
+export const TextFieldContainer = styled.div`
+  position: relative;
+`
+
+export const NodeContainer = styled.div<{ left?: boolean }>`
+  position: absolute;
+  display: flex;
+  align-items: center;
+  height: 50px;
+  z-index: 2;
+  top: 0;
+  padding: 0 ${sizes(4)};
+  ${({ left }) => (left ? 'left: 0px' : 'right: 0px')};
```

Comment:
Let's try to avoid having any styles as a string as this will not be properly parsed by our tooling, including Stylelint. You can do it like this for example:

```tsx
type PlacementProps = {
  placement: 'left' | 'right'
}

const nodePlacementStyles = ({ placement }: PlacementProps) => placement === 'left' ? css`left: 0;` : css`right: 0;`

export const NodeContainer = styled.div<PlacementProps>`
  // ...
  ${nodePlacementStyles};
`
```